h/app.py

```diff
@@ -94,6 +121,13 @@ def includeme(config):
         },
     })
 
+    # After commit events
+    config.add_request_method(EventQueue, name='after_commit_events', reify=True)
```

Comment:
Sure. I still need to put the list on the request though for storing the events for the life-time of one request. Unless there is another way to handle that?

How about keeping it like that (or renaming to `after_commit_event_queue`) and then adding a new `notify_after_commit` method to the request which basically does `request.after_commit_event_queue.append(...)`. Then we have a nicer API but can still rely on Pyramid handling the storage in memory for each request.